.gitignore

```diff
@@ -0,0 +1,321 @@
+# ignore IDE/hidden/OS cache files

```

Comment:
This will help you to prevent your private files not being automatically pushed  by Git.